src/exercism/ResistorColorTrio.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Arrays;
+
+class ResistorColorTrio {
+    // If user input less than 3 colors, throw an exception
+    String label(String[] colors) {
+        if (colors.length < 3) {
+            throw new IllegalArgumentException("User input is less than 3 colors");
+        }
+
+        int firstDigit = -1;
+        int secondDigit = -1;
+        int numberOfZero = -1;
+        String[] COLOR_LIST = {"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"};
```

Comment:
Very good solution so far. Please add some spice and create a Color Class above with a constructor that accepts `String color`. The Color class should have a `String getColour()` method that you should use in this method.

Then, Instead of `String[] COLOR_LIST`, you should have `Color[] COLOR_LIST`

Reply:
when you say "String color", you meant user inputs or "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"?